openmls/src/credentials/mod.rs

```diff
@@ -55,13 +74,18 @@ impl TryFrom<u16> for CredentialType {
     }
 }
 
-/// Struct containing an X509 certificate chain, as per Spec.
+/// X509 Certificate.
+///
+/// This struct contains an X509 certificate chain.
+/// TODO: #134 X509 certificates are not supported yet by OpenMLS.
```

Comment:
I tried to avoid surfacing the TODO in the docs. Instead, I mentioned that it was not supported yet and kept the TODO in a regular comment.